riocli/jsonschema/schemas/deployment-schema.yaml

```diff
@@ -164,6 +164,23 @@ definitions:
                 items:
                   "$ref": "#/definitions/deviceROSBagJobSpec"
 
+              features:
```

Comment:
suggestion: Can you also update the example manifests with this new field and comments?